index.html

```diff
@@ -2,61 +2,190 @@
+                        <div class="offers-icon">
+                            <svg width="100" height="68" viewBox="0 0 100 68" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path d="M16.6667 9h75V.6667h-75c-4.5834 0-8.3334 3.75-8.3334 8.3333v45.8333H0v12.5h58.3333v-12.5H16.6667V9zm79.1666 8.3333h-25c-2.2916 0-4.1666 1.875-4.1666 4.1667v41.6667c0 2.2916 1.875 4.1666 4.1666 4.1666h25c2.2917 0 4.1667-1.875 4.1667-4.1666V21.5c0-2.2917-1.875-4.1667-4.1667-4.1667zm-4.1666 37.5H75V25.6667h16.6667v29.1666z" fill="#3077C6"/>
+                            </svg>
+                        </div>
+                        <div class="offers-text">Book online</div>
+                    </div>
+                    <div class="offers-item">
+                        <div class="offers-icon">
+                            <svg width="92" height="84" viewBox="0 0 92 84" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path d="M83.4998 29.3333H57.2082l3.9583-19.0416.125-1.3334c0-1.7083-.7083-3.2916-1.8333-4.4166L55.0415.1667 27.6248 27.625c-1.5416 1.5-2.4583 3.5833-2.4583 5.875v41.6667c0 4.5833 3.75 8.3333 8.3333 8.3333h37.5c3.4584 0 6.4167-2.0833 7.6667-5.0833l12.5833-29.375c.375-.9584.5834-1.9584.5834-3.0417v-8.3333c0-4.5834-3.75-8.3334-8.3334-8.3334zm0 16.6667l-12.5 29.1667h-37.5V33.5l18.0834-18.0833-4.625 22.25h36.5416V46zM.1665 33.5h16.6667v50H.1665v-50z" fill="#3077C6"/>
+                            </svg>
+                        </div>
+                        <div class="offers-text">Real guest reviews</div>
+                    </div>
+                </div>
             </div>
         </div>
     </div>
 </div>
 <footer class="footer">
-    <div class="footer-inner">
-        <div class="footer-card">
-            <h4>About</h4>
-            <p><a href="/">How Triphouse works</a></p>
-            <p><a href="/">Careers</a></p>
-            <p><a href="/">Privacy</a></p>
-            <p><a href="/">Terms</a></p>
-        </div>
-        <div class="footer-card">
-            <h4>Property types</h4>
-            <p><a href="/">Guest houses</a></p>
-            <p><a href="/">Hotels</a></p>
-            <p><a href="/">Apartments</a></p>
-            <p><a href="/">Villas</a></p>
-            <p><a href="/">Holiday homes</a></p>
-            <p><a href="/">Hostels</a></p>
-        </div>
-        <div class="footer-card">
-            <h4>Support</h4>
-            <p><a href="/">Contact Customer Service</a></p>
-            <p><a href="/">FAQ</a></p>
-        </div>
-    </div>
-    <div class="footer-bottom">
-        <div class="copyright">
-            <p>© 2020 Triphouse, Inc. All rights reserved</p>
+    <div class="wrapper">
+        <div class="container">
+            <div class="footer-logo">
+                <img class="footer-logo-item" src="images/logo-footer.svg" alt="TripHouse">
+            </div>
+            <div class="footer-inner">
```

Comment:
![image](https://user-images.githubusercontent.com/36200303/111767087-574d2b80-88b7-11eb-9baf-c6655bf4bb67.png)
there is a point on small screen when your footer looks like this. it is not taking up the whole empty space for some reason. please rethink this